docs/docs/understand/key_concepts.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Key Concepts
+sidebar_position: 05
+---
+# Key Concepts
+
+## Providers
+
+Providers are Minder’s integration points with external systems. These enable Minder to monitor and
+secure supply chain components by fetching data, notifying Minder of changes, and mapping external
+systems to Minder’s internal ontology.
+
+Examples:
+
+* GitHub and GitLab: Track repositories, pull requests, and CI/CD pipelines.
+
+* Docker Hub: Monitor container images and their metadata.
+
+Providers communicate with Minder through APIs, webhook events, and scheduled updates. This ensures
+continuous monitoring and up-to-date information about the entities they manage.
+
+## Entities and Checkpoints
+
+Entities are representations of key components in the supply chain, such as repositories, pull requests, or artifacts.
+They allow Minder to monitor and evaluate security practices over time.
+
+**Checkpoints**:
+
+Checkpoints are points in time that capture the state of an entity. These snapshots enable:
+
+* Auditability: Tracking changes and compliance over time.
+
+* Granular Analysis: Evaluating an entity’s state at specific moments, such as a commit hash or artifact digest.
+
+**Entity Lifecycle**:
+
+1. *Registration*: An entity is registered through its provider.
+
+2. *Evaluation*: Minder evaluates the entity against defined policies.
+
+3. *Remediation*: Issues identified during evaluation are flagged or resolved.
```

Comment:
Should this be called Actions or perhaps add Alerting too to the list?

Reply:
Changed, wdyt?